SEED-platform/seed
# #2374 fix(data_importer): handle xlsx missing headers gracefully

#### Any background context you want to provide?
See issue description
#### What's this PR do?
Addresses linked issue for spreadsheets (ie xlsx files)
#### How should this be manually tested?
Upload `seed/data_importer/tests/data/example-data-properties-missing-headers.xlsx`. You should see an error message rather than the server crash and hanging response.
#### What are the relevant tickets?
#2374 
#### Screenshots (if appropriate)
<img width="599" alt="Screen Shot 2020-10-14 at 1 37 11 PM" src="https://user-images.githubusercontent.com/18518728/96037777-7e449480-0e23-11eb-9a81-0e5fafa1c2b7.png">
